src/components/TDViewer/components/InteractionSection.tsx

```diff
@@ -168,19 +196,176 @@ const InteractionSection: React.FC<IInteractionSectionProps> = (props) => {
     );
   };
 
+  const handleOnClickSendRequest = async (item: {
+    [key: string]: any;
+  }): Promise<{ value: string; error: string }> => {
+    const index = extractIndexFromId(item.id);
+    let value: any;
+
+    try {
+      const res = await readProperty(td, item.propName, undefined);
```

Comment:
The index should be given as a parameter otherwise we cannot know which one node-wot will pick (generally the first one). See https://w3c.github.io/wot-scripting-api/#dom-consumedthing-readproperty where formindex is explained as an option